scripts/codegen-firecracker.sh

```diff
@@ -27,7 +27,7 @@ cd $KANI_DIR/firecracker/src/devices/src/virtio/
 export RUSTC_LOG=error
 export RUST_BACKTRACE=1
 # Kani cannot locate Cargo.toml correctly: https://github.com/model-checking/kani/issues/717
-cargo kani --only-codegen --no-config-toml
+cargo kani --only-codegen
```

Comment:
We do still. I changed the behavior to not fail if it can't find `./Config.toml` but there's actually a todo in the code where we should switch to using locate-project.